app/views/properties/index.html.erb

```diff
@@ -43,6 +43,9 @@
               <p align="center">Price: <%= number_to_currency(property["listPrice"]) %> </p>
               <p align="center">Address: <%= property["address"]["full"] %></p>
               <p align="center">Bedrooms: <%= property["property"]["bedrooms"] %> | Full: <%= property["property"]["bathsFull"] %> Half: <%= property["property"]["bathsHalf"] %></p>
+              <p align="center">
```

Comment:
We'll be changing this so that we center things with css, Bootstrap has a class called 'text-center' for future reference